Mathlib/Algebra/Order/GroupWithZero/WithZero.lean

```diff
@@ -78,3 +77,37 @@ instance {α : Type*} [Mul α] [Preorder α] [MulRightMono α] :
         dsimp only at h ⊢
         norm_cast at h ⊢
         exact mul_le_mul_right' h x
+
+section Units
+
+variable {α : Type*} [LinearOrderedCommGroupWithZero α]
+
+open WithZero
+
+/-- Given any linearly ordered commutative group with zero `α`, this is the order isomorphism
+between `WithZero αˣ` with `α`. -/
+@[simps!]
+def withZeroUnits_OrderIso : WithZero αˣ ≃o α where
```

Comment:
```suggestion
def OrderIso.withZeroUnits : WithZero αˣ ≃o α where
```

to match [OrderMonoidIso.withZeroUnits](https://leanprover-community.github.io/mathlib4_docs/Mathlib/GroupTheory/ArchimedeanDensely.html#OrderMonoidIso.withZeroUnits)

Do you think this file's imports can be changed so that we could make the bundled OrderMonoidIso directly? We already have that it is a MulEquiv from withZeroUnitsEquiv.

Reply:
I think the best is to open a PR and see if git complains. If not, I agree that what you suggest is the best option.

Reply:
Have you checked using `#find_home`? Because I would have moved it somewhere inside `Algebra/Order/GroupWithZero'.